### DISCUSSION TITLE
Update stuck at "Starting..."

### DISCUSSION THREAD

**Discussion by marcus-at-localhost** [atrocore/atrocore#858] [Q&A] [answered]
Hi,
we rented a module and installed it on our dev instance, and it installed fine, along with all the other updates.
Now on our productive system, I did all system updates first (Core, PIM, DAM etc.) and then added the module and pressed update, and it's stuck at "Starting...".
I deleted the `data/composer.log` file and I can login again, but I can't install the module. Is there anything else I can try?
Thank you!
m.

**Comment by rratsun** [comment, marked as answer]
Hello,

Please, reboot the server. It's will help.
